AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTransmissionTargetTests.m

```diff
@@ -840,6 +853,7 @@ - (void)testOverridingCommonSchemaPropertiesWithTwoChildrenUnderTheSameParent {
 
   // Set log1.
```

Comment:
Can the author of this test provide some context? @jaelim-ms I don't know if you wrote it initially, but I think you may have? It was weird to adapt because I don't know what it was testing exactly. Who initially would have called "trackEvent"? Since events can only originate from one target, it wasn't clear to me who would "own" the properties. (Multiple transmission targets for one log is a future scenario that the PR doesn't support for now.)